lib/elixir/lib/task.ex

```diff
@@ -341,8 +341,17 @@ defmodule Task do
   Each `enumerable` item is passed as argument to the given function `fun` and
   processed by its own task. The tasks will be linked to the current process,
   similarly to `async/1`.
-
-  See `async_stream/5` for discussion, options, and examples.
+  
+  ## Example
+  
+  Count the graphemes in each string asynchronously, then add the counts together using reduce.
+  
+      iex> strings = ["long string", "longer string", "there are many of these"]
+      iex> stream = Task.async_stream(strings, fn text -> text |> String.graphemes |> Enum.count end)
```

Comment:
Alternatively we can use `String.codepoints + length()` since there is no "fast" equivalent for it.